src/main.rs

```diff
@@ -144,32 +145,48 @@ fn launch_git_with_self_as_editor() {
     run_cmd(Command::new("git").arg("commit").env("GIT_EDITOR", self_path))
 }
 
-fn collect_information_and_write_to_file(out_path: PathBuf) {
-    let maybe_emoji = select_emoji();
-
-    if maybe_emoji == None {
-        abort();
+fn git_message_is_empty(git_commit_contents: & String) -> bool {
+    for line in git_commit_contents.split("\n") {
+        if !line.starts_with('#') && line.len() > 0  {
```

Comment:
Clippy recommends using `is_empty` instead of comparing to `0`, and I can agree that it's more clear:

```suggestion
        if !line.starts_with('#') && !line.is_empty()  {
```